examples/texture/test/texture_test.dart

```diff
@@ -11,6 +11,6 @@ void main() {
         .main(); // builds the app and schedules a frame but doesn't trigger one
     await tester.pump(); // triggers a frame
 
-    expect(find.text('Fluter Blue'), findsOneWidget);
```

Comment:
Actually, why was this test not failing or is not failing now? Is it potentially not running on CI?

Reply:
Potential fix to run all tests: https://github.com/flutter/flutter/pull/138374